Tools/PowershellModule/src/Library/Get-WinGetManifest.ps1

```diff
@@ -34,7 +34,7 @@ Function Get-WinGetManifest
     Name of the Azure Function Name that contains the Windows Package Manager Rest APIs.
 
     .PARAMETER ManifestIdentifier
-    [Optional] The Windows Package Manager Package Identifier of a specific Manifest result
+    [Optional] The Windows Package Manager Manifest Identifier of a specific Application Manifest result.
```

Comment:
Package Identifier. there is no such thing as a Manifest Identifier

Reply:
This is in relation to the Parameter. This change can't be performed unless we change the Parameter name

Reply:
No. the description telling someone to provide data that doesnt exist is confusing and pointless. We need to indicate in the description that the data provided to this misnamed parameter is the Package identifier.